homeassistant/components/mqtt/strings.json

```
@@ -627,6 +650,36 @@
               }
             }
           }
+        },
+        "export": {
+          "title": "Export MQTT device config",
+          "description": "An export allows you to migrate the MQTT device configuration to YAML based configuration or MQTT discovery. The configuration export can also be helpful for troubleshooting.",
+          "menu_options": {
+            "export_discovery": "Export MQTT discovery information",
+            "export_yaml": "Export to YAML configuration"
+          }
+        },
+        "export_yaml": {
+          "title": "[%key:component::mqtt::config_subentries::device::step::export::title%]",
+          "description": "You can copy the configuration below and place it your configuration.yaml file. Home Assistant will detect if the MQTT device was tried to set up via YAML instead, and will offer a repair flow to clean up the redundant subentry.",
```

"was tried to set up" is not correct English grammar, suggestion below:

```suggestion
          "description": "You can copy the configuration below and place it your configuration.yaml file. Home Assistant will detect if the setup of the MQTT device was tried via YAML instead, and will offer a repair flow to clean up the redundant subentry.",
```

But perhaps you meant that differently …